--- a/tests/amd_smi_test/functional/hw_topology_read.cc
+++ b/tests/amd_smi_test/functional/hw_topology_read.cc
@@ -20,6 +20,7 @@
  * THE SOFTWARE.
  */

+#include <iomanip>
 #include <cinttypes>
 #include <cstdint>

--- a/tests/amd_smi_test/functional/frequencies_read.cc
+++ b/tests/amd_smi_test/functional/frequencies_read.cc
@@ -20,6 +20,7 @@
  * THE SOFTWARE.
  */

+#include <iomanip>
 #include <cstdint>
 #include <iostream>
 #include <string>

--- a/tests/amd_smi_test/functional/computepartition_read_write.cc
+++ n/tests/amd_smi_test/functional/computepartition_read_write.cc
@@ -20,6 +20,7 @@
  * THE SOFTWARE.
  */

+#include <iomanip>
 #include <cstdint>
 #include <unistd.h>

--- a/tests/amd_smi_test/test_base.cc
+++ b/tests/amd_smi_test/test_base.cc
@@ -20,6 +20,7 @@
  * THE SOFTWARE.
  */

+#include <iomanip>
 #include <gtest/gtest.h>

 #include <cassert>
